.github/workflows/generate-measurements.yml

```diff
@@ -72,23 +89,83 @@ jobs:
           azureResourceGroup: ${{ steps.az_resource_group_gen.outputs.res_group_name }}
           osImage: ${{ github.event.inputs.osImage }}
           isDebugImage: ${{ github.event.inputs.isDebugImage }}
-          cosignPublicKey: ${{ startsWith(github.ref, 'refs/heads/release/v') && secrets.COSIGN_PUBLIC_KEY || secrets.COSIGN_DEV_PUBLIC_KEY }}
-          cosignPrivateKey: ${{ startsWith(github.ref, 'refs/heads/release/v') && secrets.COSIGN_PRIVATE_KEY || secrets.COSIGN_DEV_PRIVATE_KEY }}
-          cosignPassword: ${{ startsWith(github.ref, 'refs/heads/release/v') && secrets.COSIGN_PASSWORD || secrets.COSIGN_DEV_PASSWORD }}
-          awsAccessKeyID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          awsDefaultRegion: ${{ secrets.AWS_DEFAULT_REGION }}
-          awsBucketName: ${{ secrets.PUBLIC_BUCKET_NAME }}
+          test: "nop"
+
+      # Check /docs/secure_software_distribution.md#sign-measurements
```

Comment:
This file does not exist

Reply:
Good thing it was there all the time, that's copy-paste :) 
Will remove it.